extrinsics_calib_params:
  odom_noise: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1] #Noises, rotations in radians, then translations in m
  landmark_noise: [1.0, 1.0, 1.0, 1, 1, 1]
  intrinsics_noise: [0.1, 0.1, 0.001, 0.1, 0.1]
  # extrinsics_noise: [0.1,0.1,0.1, 0.5,0.5,0.5]
  extrinsics_noise: [1,1,1, 10,10,10]
  intrinsics_guess: [530.132844320014, 528.4231401659118, 0, 484.8306373138666, 266.7362347663899] #fx, fy, skew, cx, cy
  extrinsics_guess: [1,0,0,0,  0,1,0,0, 0,0,1,0, 0,0,0,1]
  measurement_noise: 4 #Noise in pixels of a detected tag corner
  bag_file: /media/data/bagfiles/kinect_one/camera_movement_1.bag
  image_topic: /kinect2/qhd/image_color_rect
  odom_topic: /kinect_one/vicon_odom

apriltag_tracker_params:
  tag_size: 0.168
  id: 27
  K: [530.132844320014, 0.0, 484.8306373138666, 0.0, 528.4231401659118, 266.7362347663899, 0.0, 0.0, 1.0]
  D: [0,0,0,0]
  # D: [0.05137893228292329, -0.07913358530157011, -0.000939186223249489, 0.0005432608595405252, 0.028821551963704763]
  do_debug: true